account_banking_sepa_direct_debit/models/account_payment_order.py

```diff
@@ -281,18 +278,19 @@ def generated2uploaded(self):
             to_expire_mandates = abmo.browse([])
             first_mandates = abmo.browse([])
             all_mandates = abmo.browse([])
-            for bline in order.bank_line_ids:
-                if bline.mandate_id in all_mandates:
+            for payment in order.payment_ids:
```

Comment:
@pedrobaeza This field `payment_ids` doesn't exist on `account.payment.order` model.
Or maybe there is a missing dependency?

Reply:
This PR adds such field: https://github.com/OCA/bank-payment/pull/979/files#diff-e1bae20141deca080d6db095e9874359ec1ed9d92f0b4d46aedc9f3ecea71cf0R80